### PR TITLE
Docs for Number Field Plugin #726

### PR DESCRIPTION
**Pull request by ncphillips**
Docs for tinacms/tinacms#726

### CODE REVIEW

File: content/docs/fields/number.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+interface TextConfig {
+  name: string
+  component: 'number'
+  label?: string
+  description?: string
+  step?: string
+}
+```
+
+-->
```
Comment (PaulBunker) [inline content/docs/fields/number.md]: Is this an intentional arrow? Also looks like next and previous is broken.

**Comment by DirtyF** [comment]
Nice, here's an image for the number field if it hellps 😊 

number-field

File: content/docs/fields/number.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+consumes:
+  - file: /packages/tinacms/src/plugins/fields/NumberFieldPlugin.tsx
+    details: Shows text field interface and config options
+  - file: /packages/@tinacms/fields/src/NumberField.ts
+    details: Shows text field interface and config options
+---
+
+The `number` field represents a number input.
+
+<!-- TODO: Provide image of number field -->
```
Comment (DirtyF) [inline content/docs/fields/number.md]: submitted one in comment :)

File: content/docs/fields/number.md (hunk 1 above)
Comment (ncphillips) [reply content/docs/fields/number.md]: Nope! good catch

**Comment by ncphillips** [comment]
Okay, I think I fixed up those issues. Thanks @DirtyF and @PaulBunker

File: content/docs/fields/number.md (hunk 1 above)
Comment (PaulBunker) [reply content/docs/fields/number.md]: It actually looked quite pretty on the preview site 😂

**Pull request merged**